OpenShift.md

```diff
@@ -4,14 +4,9 @@ This example will use [minishift](https://docs.openshift.org/latest/minishift/ge
 First step is to start this environment:
 
 ```
-minishift start --openshift-version=v3.6.0-alpha.2
+minishift start
 ```
 
-You can use a more [recent version](https://github.com/openshift/origin/releases) if available.
-
-NOTE: The alpha version is currently required, as the previous stable version does not include Kubernetes RBAC authorizaion
-support.
-
 When fully started, then launch the dashboard using the link displayed following `The server is accessible via web console at:
 `. Log in using the credentials `developer/developer` and navigate to the `New Project` overview page.
 
```

Comment:
The next line contains "Kubernetes", could you change this to "OpenShift" - looks like I missed that (copy/paste) :)